components/Date.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export function Date(props) {
+  const { id, label, date } = props
+  const dateFormatted = date ? date.replace(/^(.{4})(.{2})/gm, '$1-$2-') : 'NA'
+  return (
+    <dl id={id} data-testid={id} className="ds-mt-8 ds-py-2 ds-font-body">
```

Comment:
You will need to remove the ds prefix here for the classes

Reply:
Good catch